components/cli/pkg/commands/setup_create_existing_cluster.go

```diff
@@ -103,7 +103,7 @@ func RunSetupCreateOnExistingCluster(isPersistentVolume, hasNfsStorage, isLoadBa
 		isLoadBalancerIngressMode, nfs, db); err != nil {
 		util.ExitWithErrorMessage("Failed to deploy cellery runtime", err)
 	}
-	util.WaitForRuntime(false, false)
+	runtime.WaitForRuntime(false, false)
```

Comment:
Shall we change this to `runtime.WaitFor`?

Reply:
Updated.